ophyd/signal.py

```diff
@@ -397,6 +397,21 @@ def _run_metadata_callbacks(self):
                                       **self._metadata)
 
 
+class SignalRO(Signal):
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self._metadata.update(
+            connected=True,
+            write_access=False,
+        )
+
+    def put(self, value, *, timestamp=None, force=False):
+        raise ReadOnlyError("The signal {} is readonly.".format(self.name))
+
+    def set(self, value, *, timestamp=None, force=False):
+        raise ReadOnlyError("The signal {} is readonly.".format(self.name))
```

Comment:
Can it be implemented as placeholders, something like the snippet below?
```py
    def put(self, *args, **kwargs):
         raise ReadOnlyError("The signal {} is readonly.".format(self.name))

    def set(self, *args, **kwargs):
        self.put(*args, **kwargs)
```

This way one does not need to sync error messages in multiple places. @hhslepicka, @danielballan, @klauer?

Reply:
`set = put`

Reply:
That works for me too.

Reply:
I will just merge it now and we can discuss that later on...

Reply:
Sounds good.